src/common/types.cpp

```diff
@@ -1150,6 +1150,19 @@ const string EnumType::GetValue(const Value &val) {
 	}
 }
 
+idx_t EnumType::GetValuePosition(const Value &val) {
+	switch (val.type().InternalType()) {
+	case PhysicalType::UINT8:
+		return val.value_.utinyint;
```

Comment:
`val.GetValue<uint32_t>()` should cover all these three cases so the switch is no longer required

Reply:
it doesn't but i've moved it there.